examples/morpheusvm/controller/controller.go

```diff
@@ -28,28 +28,28 @@ var (
 )
 
 // New returns a VM with the indexer, websocket, and rpc apis enabled.
-func New(options ...vm.Option) *vm.VM {
-	opts := []vm.Option{
-		indexer.WithIndexer(consts.Name, indexer.Endpoint),
-		ws.WithWebsocketAPI(10_000_000),
-		vm.WithVMAPIs(jsonrpc.JSONRPCServerFactory{}),
-		vm.WithControllerAPIs(&jsonRPCServerFactory{}),
+func New(namespacedOptions ...vm.NamespacedOption) *vm.VM {
```

Comment:
Since we are only going to have one option type, we should just leave the name as `vm.Option`